packages/backend/src/functions/ceremony.ts

```diff
@@ -55,3 +75,260 @@ export const stopCeremony = functions.pubsub.schedule(`every 30 minutes`).onRun(
         })
     }
 })
+
+/**
+ * Register all ceremony setup-related documents on the Firestore database.
+ * @dev this function will create a new document in the `ceremonies` collection and as needed `circuit`
+ * documents in the sub-collection.
+ */
+export const setupCeremony = functions.https.onCall(
```

Comment:
should we rename this to setupPhase2Ceremony? given we are setting the properties in Firestore as Phase2

Reply:
IMHO, I would suggest no for two main reasons: 
1) We are not supporting any Phase 1 right now.
2) When we support the Phase 1, we are going to use a flag to discriminate (e.g., `p0tion setup --phase 1 (or 2)).

Reply:
sounds good